README.md

```diff
@@ -100,3 +100,11 @@ Below is a visual sample of data schema. For the latest actual data please refer
   }
 ]
 ```
+
+## Data Sources and Intended Use
+
+At the moment the data sources used to generate the [speed data](default_speeds.json) are [Mapillary](https://mapillary.com) for GPS traces and [OSM](https://osm.org) to which the traces are matched (via the [Valhalla](https://github.com/valhalla/valhalla routing APIs)). These are the only two sources that are currently supported in the [conflation](https://github.com/OpenStreetMapSpeeds/conflation) tool which is tasked with generating this data automatically. Mapillary has been a huge supporter of OpenStreetMap since it's inception and allows its APIs to be used to improve OSM (see section 13 of their [terms](https://www.mapillary.com/terms)). This allows us to generate "default" speed values for different parts of the world and make that information readily available along side and for use with the OSM dataset. Since adding the data directly to OSM would likely be considered vandalism (but also a huge waste of space) we will instead opt to link this data artifact either on an existing OSM wiki page (eg. https://wiki.openstreetmap.org/wiki/Average_speed_per_way, https://wiki.openstreetmap.org/wiki/Proposed_features/Practical_maxspeed) or create a new one and advertize it to folks looking for sensible defaults. One example of a place these defaults could be useful would be in any OSM-based router, and indeed the Valhalla router currently supports this data format for specifying default motor vehicle speeds in its graph creation.
+
+Thus far, the data currently available in this repository has been manually generated. As such, the data is coarse both in terms of the indivual speeds (5kph increments) but also in terms of their geographic specificity (global and a few countries exist). The primary tools used in generating these values by-hand were Mapillary and its APIs as well as [Overpass Turbo](https://overpass-turbo.eu/) (which is a query API on top of OSM). The steps are basically to use Overpass Turbo to find ways that match the criteria of the part of the schema you are trying to fill in the geography you are trying to fill it. Once you've found some ways you need to find where they are covered in Mapillary's map. Once you've done that you can use the Mapillary API's to get the timestamps when a person got on and off of the way you are interested in. Finally you need to get the length of the way, which you can do with a router either directly on osm.org or with whatever OSM-based router you have access to. To speed up the process, especially for those types of ways who have multiple highway tags, one can just get estimates of the motorway, trunk and service speeds and do a linear interpolation for the rest of the speeds between trunk and service.
```

Comment:
I think there's a word missing / some extra words here? `The steps are basically to use Overpass Turbo to find ways that match the criteria of the part of the schema you are trying to fill in the geography you are trying to fill it.`

Reply:
ill reword, its probably m issing commas :smile: